server/src/grpc_server.rs

```diff
@@ -76,10 +85,24 @@ impl<OB: QuoteBook> GrpcServer<OB> {
 
         let health_service =
             mc_util_grpc::HealthService::new(None, self.logger.clone()).into_service();
+        let quote_minimum_map = self.quote_minimum_map.clone();
+
+        let sci_validator: SciValidator = Arc::new(move |sci| {
+            let base_token_id = TokenId::from(sci.pseudo_output_amount.token_id);
+            let base_amount = sci.pseudo_output_amount.value;
+            let minimum_for_token_id = *quote_minimum_map.get(&base_token_id).unwrap_or(&0);
+            if base_amount < minimum_for_token_id {
+                return Err(QuoteBookError::UnsupportedSci(
+                    format!("Quote is too small for deqs. Quotes with base_token: {} require a minimum of: {}", base_token_id, minimum_for_token_id
```

Comment:
```suggestion
                    format!("Quote volume is too small for deqs. Quotes with base_token: {} require a minimum of: {}", base_token_id, minimum_for_token_id
```

Reply:
So this will only apply to quotes that come from the grpc interface, right? and not quotes that come from peers

that seems fine for now imo